includes/forum-database.php

```diff
@@ -160,6 +162,17 @@ public function buildDatabase() {
             KEY parent_id_id (parent_id, id)
             ) $charset_collate;";
 
+            $sql[] = 'CREATE TABLE '.$this->tables->edit_history." (
+            id int(11) NOT NULL auto_increment,
+            post_id int(11) NOT NULL default '0',
+            user_id int(11) NOT NULL default '0',
```

Comment:
integer display width was never a good idea anyway, and has been deprecated in MySQL8, quote

>Warning: #1681 Integer display width is deprecated and will be removed in a future release.

for more info see http://www.tocker.ca/2015/07/02/proposal-to-deprecate-mysql-integer-display-width-and-zerofill.html

```suggestion
            id int NOT NULL auto_increment,
            post_id int NOT NULL default '0',
            user_id int NOT NULL default '0',
```